### DISCUSSION TITLE
Change font color

### DISCUSSION THREAD

**Discussion by jai-cs** [0x7c13/Notepads#880] [Q&A] [answered]
First of all amazing application and I am already in love with it. Just wanted to ask if there a is an option to change font color since I did not find one. If not will it be implement anytime soon ?

**Comment by 0x7c13** [comment, marked as answer]
There is no plan to add option to change font color as of now.

**Reply by HubKing** [reply]
I only need a few features for a text editor, and this application fits almost all of them except its inability to change editing area colours. I need higher contrast like white text on black background. All text editors I know of, except Windows Notepad, at least support changing font/background colours for the editing area. I think it is one of the basic features that must be present.

**Reply by hello-smile6** [reply]
> I only need a few features for a text editor, and this application fits almost all of them except its inability to change editing area colours. I need higher contrast like white text on black background. All text editors I know of, except Windows Notepad, at least support changing font/background colours for the editing area. I think it is one of the basic features that must be present.

It should be possible to implement ANSI escape sequences in a text editor, which would allow colored text.

**Comment by AdrielXXO** [comment]
This program really need a change color for bars. Like fluent terminal or terminal Preview. Its a really good idea!
